vita3k/gui/src/settings_dialog.cpp

```diff
@@ -853,18 +853,18 @@ void draw_settings_dialog(GuiState &gui, EmuEnvState &emuenv) {
             ImGui::Spacing();
             ImGui::SliderFloat("Background Alpha", &emuenv.cfg.background_alpha, 0.999f, 0.000f);
             if (ImGui::IsItemHovered())
-                ImGui::SetTooltip("Select your preferred transparent background effect.\nThe minimum slider is opaque and the maximum is transparent.");
+                ImGui::SetTooltip("Select your preferred transparenu of the background.\nThe minimum slider is opaque and the maximum is transparent.");
```

Comment:
are you sure there is no mistypes?

Reply:
fixed, I'll go over the changes once more